hack/schemapatch-config.yaml

```diff
@@ -280,6 +280,7 @@ k8s.io/api/core/v1.ResourceRequirementsMask:
     - Requests
 k8s.io/api/core/v1.SecurityContext:
   fieldMask:
+    - AllowPrivilegeEscalation
```

Comment:
@evankanderson I didn't realize y'all had added this, so I totally missed it.  FYI in case you are adding seccomp stuff, you may need this too (if you were following my bad example! 😂 )

Reply:
@markusthoemmes added this, so that we could produce a structural schema for our crds.

Reply:
It was just after my time.  TIL!